awx/main/migrations/0018_v330_create_user_session_membership.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
```

Comment:
These migrations will have to roll forward numerically and this one will need to have its dependencies updated.